src/utils/DeploymentBuilder.ts

```diff
@@ -118,15 +116,15 @@ export class DeploymentBuilder {
 
 type BuildEntityInternalOptions = {
   hashesByKey?: Map<string, ContentFileHash>
-  filesByHash?: Map<ContentFileHash, Buffer>
+  filesByHash?: Map<ContentFileHash, Uint8Array>
   metadata?: EntityMetadata
   timestamp?: Timestamp
 }
 
 /** This data contains everything necessary for the user to sign, so that then a deployment can be executed */
 export type DeploymentPreparationData = {
   entityId: EntityId
-  files: Map<ContentFileHash, Buffer>
+  files: Map<ContentFileHash, Uint8Array>
```

Comment:
This is the only breaking change. Buffer is assignable to Uint8Array but not the other way around